README.md

```diff
@@ -1,2 +1,56 @@
 # structlog-sentry
-SentryProcessor for structlog
+
+| What          | Where                                         |
+| ------------- | --------------------------------------------- |
+| Documentation | <https://github.com/kiwicom/structlog-sentry> |
+| Maintainer    | @paveldedik                                   |
+
+Based on <https://gist.github.com/hynek/a1f3f92d57071ebc5b91>
+
+## Usage
+
+This module is intended to be used with `structlog` like this:
+
+```python
+import structlog
+from structlog_sentry import SentryProcessor
+
+
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_log_level,  # required before SentryProcessor()
+        SentryProcessor(sentry_log_level=logging.WARNING),
+    ],
+    logger_factory=...,
+    wrapper_class=...,
+)
+
+
+log = structlog.get_logger()
+
+try:
+    1/0
+except ZeroDivisionError:
+    log.error()  # automatically logged in Sentry
+```
+
+Do not forget to add the `structlog.stdlib.add_log_level` processor before
+`SentryProcessor`. The `SentryProcessor` class takes the following arguments:
+
+- `sentry_log_level` - logging level of events to be reported in Sentry
```

Comment:
"events of this or higher levels will be reported to Sentry"

Reply:
Thanks, that's the correct way to say it.